### DISCUSSION TITLE
where do i download presets?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [asb2m10/dexed#418] [General]
where do i get dexed presets?

**Comment by SamuelSHrk** [comment]
http://bobbyblues.recup.ch/yamaha_dx7/dx7_patches.html

**Comment by asb2m10** [comment]
A user from KVR (Jacques PRESTREAU AKA BlackWinny) did an awesome DX7 cartridge compilation.

https://hsjp.eu/downloads/Dexed/Dexed_cart_1.0.zip
